security/host-forwarded-traffic.md

```diff
@@ -42,15 +42,20 @@ In contrast, if applyOnForward is set to true for a policy that selects a HEP, t
 - Egress policy on HEP eth1 affects connections 2, 3, and 4
 
 There are also different default action semantics for **applyOnForward: true policy** versus **applyOnForward: false policy**.
-An applyOnForward: true policy affects all traffic through the HEP (connections 1-4). If no applyOnForward policy selects the HEP and direction (ingress versus egress), then forwarded traffic is allowed.  If no policy (regardless of applyOnForward) selects the HEP and direction, then local traffic is denied.
-
-| **HEP defined?** | **Traffic Type** | **applyOnForward defined?** | **Any policy defined?** | **Default Action** |
-| ---------------- | ---------------- | --------------------------- | ----------------------- | ------------------ |
-| No               | Any              | n/a                         | n/a                     | Allow              |
-| Yes              | Forwarded        | No                          | Any                     | Allow              |
-| Yes              | Forwarded        | Yes                         | Yes                     | Deny               |
-| Yes              | Local            | n/a                         | No                      | Deny               |
-| Yes              | Local            | n/a                         | Yes                     | Deny               |
+An applyOnForward: true policy affects all traffic through the HEP (connections 1-4). If no applyOnForward policy selects the HEP and direction (ingress versus egress), then forwarded traffic is allowed.  If no policy (regardless of applyOnForward) selects the HEP and direction, then local traffic is denied unless if the HEP has the `projectcalico-default-allow` profile.
+
+| **HEP defined?** | **Traffic Type** | **applyOnForward defined?** | **Any policy defined?** | **Has allow-all profile** | **Default Action** |
+| ---------------- | ---------------- | --------------------------- | ----------------------- | ------------------------- | ------------------ |
+| No               | Any              | N/A                         | N/A                     | N/A                       | Allow              |
+| Yes              | Forwarded        | No                          | Any                     | N/A                       | Allow              |
```

Comment:
Could we change Any -> Either.   I personally have difficulty parsing this because the columns name is "Any policy defined".

Reply:
addressed in 6477b33